### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by markborkum**
This pull request provides an example implementation of a Pacifica Dispatcher.

The code is heavily commented and favors readability over [PEP 8](https://www.python.org/dev/peps/pep-0008/) recommendations.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Pacifica Proxymod version
```
Comment (dmlb2000) [inline .github/ISSUE_TEMPLATE.md]: This should be fixed, wrong name `Proxymod`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,56 @@
-# pacifica-dispatcher-example
-Pacifica Dispatcher Example Repository
+# Pacifica Dispatcher Example
+[![Build Status](https://travis-ci.org/pacifica/pacifica-dispatcher-example.svg?branch=master)](https://travis-ci.org/pacifica/pacifica-dispatcher-example)
+[![Build status](https://ci.appveyor.com/api/projects/status/eg2r1y37yvxi0b5p?svg=true)](https://ci.appveyor.com/project/dmlb2000/pacifica-example)
+[![Maintainability](https://api.codeclimate.com/v1/badges/f2dba248b1a7966e5a49/maintainability)](https://codeclimate.com/github/pacifica/pacifica-dispatcher-example/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/f2dba248b1a7966e5a49/test_coverage)](https://codeclimate.com/github/pacifica/pacifica-dispatcher-example/test_coverage)
```
Comment (dmlb2000) [inline README.md]: This line can get chucked, this coverage report thing depends on out dated pipelines to report... I need to investigate getting a badge for 100% coverage that isn't code climate.

**Review by dmlb2000 — APPROVED** [review]
I'll add some travis bits to this and otherwise it looks good.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (markborkum) [reply .github/ISSUE_TEMPLATE.md]: Done!

File: README.md (hunk 2 above)
Comment (markborkum) [reply README.md]: Done!

**Pull request merged**